### PR TITLE
Add dickey_fulley LinAlg and ValueError test

### PR DESCRIPTION
**Pull request by nikhaas**

### CODE REVIEW

File: tests/feature_extraction/test_feature_calculations.py [hunk 1]
```diff
@@ -195,6 +195,14 @@ def test_augmented_dickey_fuller(self):
         self.assertLessEqual(res["attr_pvalue"], 0.05)
         self.assertEqual(res["attr_usedlag"], 0)
 
+        # Check if LinAlgError and ValueError are catched
```
Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: You could also achieve this by mocking the `adfuller` method

**Comment by MaxBenChrist** [comment]
We are getting close to 99% coverage, great @nikhaas

File: tests/feature_extraction/test_feature_calculations.py (hunk 1 above)
Comment (nikhaas) [inline tests/feature_extraction/test_feature_calculations.py]: What do you mean with "mocking"?

Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: http://www.voidspace.org.uk/python/mock/patch.html

you patch the function, change it to one that raises the error of interest, always, no madder which input

Comment (nikhaas) [inline tests/feature_extraction/test_feature_calculations.py]: Thx!

**Pull request merged**